### DISCUSSION TITLE
customtkinter.CTkToplevel() is not working

### DISCUSSION THREAD

**Discussion by Hancie123** [TomSchimansky/CustomTkinter#695] [Q&A]
def add_driver_gui():
            driver = customtkinter.CTkToplevel()
            driver_registration.DriverRegistration(driver)
            driver.mainloop()

**Comment by Akascape** [comment]
Please read the documentation for CTkTopLevel:
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkToplevel

**Reply by Hancie123** [reply]
I have read it but not working properly. What can i do now?

**Reply by Akascape** [reply]
@Hancie123 
Ig the problem is with `driver_registration.DriverRegistration(driver)`
Debug it!

**Comment by TomSchimansky** [comment]
You don't have to call .mainloop() for a TopLevel window, it just works if you called maintop on you main CTk window.
